TLM/TLM/Manager/Impl/ExtNodeManager.cs

```diff
@@ -41,11 +41,14 @@ public class ExtNodeManager
             return JunctionHasOnlyHighwayRoads(nodeId) && !LaneConnection.LaneConnectionManager.Instance.Sub.HasNodeConnections(nodeId);
         }
 
-        public GetNodeSegmentIdsEnumerable GetNodeSegmentIds(ushort nodeId, ClockDirection clockDirection) {
+        public GetNodeSegmentIdsEnumerable GetNodeSegmentIds(ushort nodeId, ClockDirection clockDirection)
+            => GetNodeSegmentIds(nodeId, clockDirection, null);
+
```

Comment:
would it make more sense to make these methods static and put them inside node extensions (`GetNodeSegmentIds(ref this NetNode  netNode, ...`)?
Also there is no need to make `initialSegmentId` nullable sense both method prototypes exit.

Reply:
I'm not opposed to that change, but it seems outside the scope of this PR. This is a change to `JunctionRestrictionsManager`--changes to other classes should be kept to the minimum necessary for functionality.

initialSegmentId is nullable because that is a convention for an unspecified argument. It allows for cleaner and more readable code than just using zero, since for example, you can use the null coalescing assignment ??=

Reply:
yeah its out of scope.

> initialSegmentId is nullable because that is a convention for an unspecified argument.

No I mean never pass null pr zero always pass something valid. if you don't want to specify initialSegmentId just call the overload that does not need the  initialSegmentId  argument.

Reply:
> No I mean never pass null pr zero always pass something valid.

You're right. I no longer remember what I was thinking, doing it that way. I probably thought I was going to do something different in the overload without `initialSegmentId`, or maybe I just wasn't thinking at all. 🤷‍♂️